rules/convencionBranchs.md

```diff
@@ -0,0 +1,85 @@
+# 🌿 Convención de Ramas y Pull Requests
+
+Este documento define el flujo de trabajo y las reglas para la creación y gestión de ramas y Pull Requests (PRs) en el proyecto.
+
+---
+
+## 🌳 1. Flujo de Ramas
+
+El flujo de trabajo se basa en **crear una rama por cada tarea**. Cada rama debe estar asociada a una corrección (`fix`), una nueva característica (`feature`) o una refactorización (`refactor`).
+
+### 🔸 Nomenclatura de Ramas
+
+Para mantener la consistencia, todas las ramas deben seguir estas reglas:
+
+- **Idioma:** Escribir nombres en **inglés**.
+- **Separador:** Usar **guiones medios** (`-`) para separar palabras.
+- **Minúsculas:** Todo el nombre debe estar en minúsculas.
+- **Autor:** Incluir el **nombre del autor** al inicio.
+- **Sin caracteres especiales:** No usar números de issues o símbolos.
+
+---
+
+### 🔹 Ejemplos de Nomenclatura
+
+| Tipo            | Ejemplo Válido                             | Ejemplo Inválido                              |
+| :-------------- | :----------------------------------------- | :-------------------------------------------- |
+| **Feature**     | `vic/reports-generator`                    | `vic-reports-generator-#3` (usa número)       |
+| **Feature**     | `azza/auth-module`                         | `Azza/AuthModule` (usa mayúsculas)            |
+| **Refactor**    | `justin/customer-management-refactor`      | `justin/customer_management` (usa guion bajo) |
+| **Alternativa** | `justin/customer-management-attempt-three` |                                               |
```

Comment:
Why don’t we use a defined standard like GitFlow? What's the advantage of using this proposed convention?

If you are creating a PR from the branch, the creator context exists. Simple and standard are better for preventing mental issues.

Maybe, if the issue is to keep the repository clean without orphan branches, it is better to use a flow of forking.